packages/dds/tree/api-report/tree.public.api.md

```diff
@@ -83,7 +83,7 @@ export interface FieldSchemaMetadata<TCustomMetadata = unknown> {
     readonly description?: string | undefined;
 }
 
-// @public
+// @public @sealed
```

Comment:
Strictly speaking, this is a breaking change. How confident are we that this will not be disruptive? If we're confident, we should at least note this change in the changeset.